Makefile

```diff
@@ -47,36 +47,46 @@ TFORMAT := short
 
 export PATH:=$(GOBIN):$(PATH)
 
-env:  ## Print Go env variables.
+env: # @HELP Print Go env variables.
+env:
 	go env
 
-deps: $(GOMAJOR)  ## List available module dependency updates.
+deps: # @HELP List available module dependency updates.
+deps: $(GOMAJOR)
 	gomajor list
 
-tparse: $(TPARSE)  ## Run all tests and outputs a coverage report using tparse.
+tparse: # @HELP Run all tests and outputs a coverage report using tparse.
```

Comment:
Nitpick: mixed tenses, should be `@HELP Run all tests and output a coverage report using tparse.`